pydis_site/apps/api/models/bot/message.py

```diff
@@ -49,6 +54,13 @@ class Message(ModelReprMixin, models.Model):
         help_text="Embeds attached to this message."
     )
 
+    @property
+    def timestamp(self) -> datetime:
+        """Attribute that represents the message timestamp as derived from the snowflake id."""
+        tz_naive_datetime = datetime.utcfromtimestamp(((self.id >> 22) + 1420070400000) / 1000)
+        tz_aware_datetime = timezone.make_aware(tz_naive_datetime, timezone=pytz.timezone("UTC"))
```

Comment:
I would prefer we don't import `pytz` here and use Django's `timezone.utc` directly, as [in its current source, those are equivalent](https://docs.djangoproject.com/en/2.2/_modules/django/utils/timezone/#make_aware). However, we don't directly include `pytz` as a dependency, and in case `pytz` is ever dropped from Django (for instance to use https://docs.python.org/3/library/datetime.html#datetime.timezone instead), we'll still be using what they allow us to. `make_aware` and `utc` are conveniently put in the same module, too.

```suggestion
        tz_aware_datetime = timezone.make_aware(tz_naive_datetime, timezone=timezone.utc)
```

Reply:
That's a good point; I'll change it.